website/docs/d/scc_account_locations.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: "ibm"
+subcategory: "Security and Compliance Center"
+page_title: "IBM : ibm_scc_account_locations"
+description: |-
+  Get information about scc_account_locations
+---
+
+# ibm_scc_account_locations
+
+Provides a read-only data source for scc_account_locations. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_scc_account_locations" "scc_account_locations" {
+}
+```
+
+
+## Attribute Reference
+
+In addition to all argument references listed, you can access the following attribute references after your data source is created.
+
+* `id` - The unique identifier of the scc_account_locations.
+* `locations` - (Required, List)
```

Comment:
Can you please remvoe required...its. a computed attribute

For all the inner attributes also